static/app/views/settings/organizationIntegrations/sentryAppExternalForm.tsx

```diff
@@ -27,8 +27,8 @@ export type FieldFromSchema = Omit<Field, 'choices' | 'type'> & {
 };
 
 export type SchemaFormConfig = {
-  description: string | null;
   uri: string;
+  description?: string | null;
```

Comment:
Doesn't seem like `description` exists in the backend:
https://github.com/getsentry/sentry/blob/c62c6e5c0bdba0e37d34a3feca13549f6846573b/src/sentry/sentry_apps/components.py#L47-L63